pkg/reconciler/taskrun/resources/taskref.go

```diff
@@ -217,13 +219,13 @@ type LocalTaskRefResolver struct {
 // return an error if it can't find an appropriate Task for any reason.
 // TODO: if we want to set source for in-cluster task, set it here.
 // https://github.com/tektoncd/pipeline/issues/5522
-func (l *LocalTaskRefResolver) GetTask(ctx context.Context, name string) (v1beta1.TaskObject, *v1beta1.ConfigSource, error) {
+func (l *LocalTaskRefResolver) GetTask(ctx context.Context, name string) (*v1beta1.Task, *v1beta1.ConfigSource, error) {
```

Comment:
Should we update the doc string here to reflect the conversion of Cluster Task to Task?

Reply:
Thanks Emma, good point! I updated the TaskObject and added the docstring at the helper function.